confignode/src/assembly/resources/conf/iotdb-confignode.properties

```diff
@@ -101,4 +101,20 @@ config_node_address_lists=host0:22278,host1:22278,host2:22278
 # data_dirs=data\\data
 # For Linux platform
 # If its prefix is "/", then the path is absolute. Otherwise, it is relative.
-# data_dirs=data/data
\ No newline at end of file
+# data_dirs=data/data
+
+####################
+### Region Configuration
+####################
+
+# The number of replicas of each region
+# Datatype: int
+# region_replica_count=3
+
+# The number of SchemaRegions of each StorageGroup
+# Datatype: int
+# schema_region_count=1
+
+# The number of DataRegions of each StorageGroup
+# Datatype: int
+# data_region_count=1
```

Comment:
Why 1?Shall we use 0 to indicate cores / region_replica_count

Reply:
This is just for 3.30 version... I will replace this parameter with minimal_data_region_number and maximal_data_region_number in April.